core/client/fs/src/main/java/alluxio/client/file/cache/store/RocksPageStore.java

```diff
@@ -151,9 +155,10 @@ public void close() {
   }
 
   private static byte[] getKeyFromPageId(PageId pageId) {
-    ByteBuffer buf = ByteBuffer.allocate(KEY_LEN);
-    buf.putLong(pageId.getFileId());
+    byte[] fileId = pageId.getFileId().getBytes();
+    ByteBuffer buf = ByteBuffer.allocate(Long.BYTES + fileId.length);
     buf.putLong(pageId.getPageIndex());
+    buf.put(fileId);
```

Comment:
Is it intentional to reverse the key order?

Reply:
yeah, it is easier to read the rest of the buffer as string when we decode the page id from byte array.